lib/utils.js

```diff
@@ -33,6 +34,15 @@ function inspectObject (args) {
   }
 }
 
+function parseExtensionArgs (extensionArgs, extensionName) {
+  const parsedExtensionArgs = _.isString(extensionArgs) ? parseJsonStringOrFile(extensionArgs) : {};
+  const extensionSpecificArgs = _.has(parsedExtensionArgs, extensionName) ? parsedExtensionArgs[extensionName] : {};
```

Comment:
same here

Reply:
it is ok to use `parsedExtensionArgs[extensionName]` here since it will be undefined if the item does not exist